### ISSUE TITLE
[Bug] - Cannot save tagline if registration mode is set to application

### ISSUE THREAD

**Issue by 0xAnansi** [LemmyNet/lemmy-ui#1523]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

When the instance is set to require registration application, even if the "Application Questionnaire" is set, when saving a tagline an error is shown saying that "application_question_required"

### Steps to Reproduce

1. Set instance "Registration Mode" to "Require registration application"
2. Fill the "Application Questionnaire" with anything
3. Save
4. Re-open settings
5. in Taglines, add anything
6. Click save
7. Error pops-up in the bottom left corner

### Technical Details

BE: 18.0
Web-browser: Firefox 114.0.2 

Request:
```json
{"taglines":["La communauté française AOP","La communauté française Label Rouge"],"auth":"eyJ_REDACTED_"}
```

Response:
```json
{"error":"application_question_required"}
```

### Lemmy Instance Version

0.18

### Lemmy Instance URL

https://jlai.lu

**Comment by alectrocute** [comment]
I've noticed this bug. Are you unable to add/edit taglines entirely?

**Comment by 0xAnansi** [comment]
> I've noticed this bug. Are you unable to add/edit taglines entirely?

Yes, this makes you unable to even click save on the current tagline page without modifying it.

**Comment by asimons04** [comment]
Adding onto this issue:

The same error happens when you try to change the rate limits as well.  Changing the registration mode to either full open or full closed and _then_ setting the rate limits works (same as with taglines).

**Comment by Nutomic** [comment]
Cant reproduce this, was probably fixed in backend.
